### ISSUE TITLE
Feature Request: ASSERT_LONG_DOUBLE_EQ, EXPECT_LONG_DOUBLE_EQ

### ISSUE THREAD

**Issue by myd7349** [google/googletest#1768]
It seems that we have `EXPECT_FLOAT_EQ` and `EXPECT_DOUBLE_EQ`, but not `EXPECT_LONG_DOUBLE_EQ`.

It may be helpful for defining similar macros for `long double`.

**Comment by myd7349** [comment]
Related issue: #890

**Comment by danger89** [comment]
Indeed. To add some more info `ASSERT_NEAR` only excepts doubles, which will loss precision.

**Comment by gennadiycivil** [comment]
Related http://go/gh/google/googletest/pull/2077

**Comment by danger89** [comment]
> e/googletest/pull/2077

https://github.com/google/googletest/pull/2077
